### PR TITLE
Issue #1 feat: Versioning Js files added with joomla custom tag method

### PR DESCRIPTION
**Pull request by lakhanmandloi**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Fixed by Sudhir. PR Sent by Lakhan.

**Comment by lakhanmandloi** [comment]
@coolbung Pls review this.

**Review by coolbung — CHANGES_REQUESTED** [review]
Why deal with only .min.js you should handle all js. Also, I guess it's possible to add CSS also via custom tag right ?

**Comment by lakhanmandloi** [comment]
@coolbung Hi, Changes made. Can you pls review it.?

File: deploytools.php [hunk 1]
```diff
@@ -68,8 +68,18 @@ public function onBeforeCompileHead() {
...
 			$head['scripts'] = $newarray;
 		}
+		if ($this->params->get('process_custom', 1)) {
+			$newarray = array();
+			foreach ($headerstuff['custom'] as $key => $value) {
+				$appendjs = ".js?".$version;
+				$appendcss = ".css?".$version;
+				$outputjs = str_replace(".js",$appendjs,$value);
+				$newarray[$str] = str_replace(".css",$appendcss,$outputjs);
+			}
```
Comment (coolbung) [inline deploytools.php]: - PHPCS missing
- Improve code readability. See if you can reduce use of variables

**Comment by lakhanmandloi** [comment]
@coolbung Hello Sir, PHPCS fixes done. Can you pls review it ?

**Pull request merged**